app/assets/stylesheets/shared_accesses.scss

```diff
@@ -5,3 +5,15 @@
     background-color: $primary-light;
   }
 }
+
+#shared-access-form {
+  input.search-bar {
+    width: 100%;
+  }
+}
+
+#shared-access-modal {
+  .modal-header {
+    margin-left: 0 !important;
+  }
+}
```

Comment:
We would need to add props for those as they are both custom components (SearchBar and Modal).
I'm on the fence about this because after reviewing the mock-ups, it seems like those two are outliers:

- The modal-header in the SharedAccess modal seems to be the only one that is not centered.
- The input in the SharedAccess form searchbar seems to be the only one that is extra wide.

So I'm not sure it's worth it to add an extra prop only for those two cases.